website/docs/guides/getting-started/ros-2.md

```diff
@@ -30,15 +30,31 @@ $ ros2 bag record -s mcap --all --storage-config-file my_storage_config.yaml
 
 ## Convert to MCAP
 
+### Using `ros2 bag convert`
+
+To convert your existing ROS 2 `.db3` files into MCAP files, you can use `ros2 bag convert` from within your ROS2 workspace.
+
+```bash
+$ cat << EOF > convert.yaml
+output_bags:
+  - uri: ros2_output
+    storage_id: mcap
+    all: true
+EOF
+$ ros2 bag convert -i ros2_input.db3 -o convert.yaml
+```
+
 ### Using the `mcap` CLI tool
 
-To convert your existing ROS 2 db3 files into MCAP files, [install the mcap CLI tool](https://github.com/foxglove/mcap/tree/main/go/cli/mcap#installing) and run the following command:
+The [`mcap` CLI tool](https://github.com/foxglove/mcap/tree/main/go/cli/mcap#installing) also supports converting ROS 2 `.db3` files into MCAP files directly.
```

Comment:
We should probably call out that if you have difficulty getting the mcap cli to discover message definitions, then try the rosbag2 method as preferred approach.